### PR TITLE
Add a test for ObjectStore\NoopScanner

### PR DESCRIPTION
**Pull request by nickvergessen**
@DeepDiver1975 @PVince81 @MorrisJobke 
Follow up for #14682 

Just creates the object and calls the methods to make sure nothing can go wrong.

### CODE REVIEW

File: tests/lib/files/objectstore/noopscanner.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+		$this->assertEquals(
+			[],
+			$this->scanner->scan('', \OC\Files\Cache\Scanner::SCAN_SHALLOW),
+			'Asserting that no error occurred while scan(SCAN_SHALLOW)'
+		);
+
+		$this->scanner->backgroundScan();
+
+		$this->scanner->backgroundScan();
```
Comment (PVince81) [inline tests/lib/files/objectstore/noopscanner.php]: twice?

Comment (nickvergessen) [inline tests/lib/files/objectstore/noopscanner.php]: Ctrl + D vs Ctrl + S,
fixed.

Comment (nickvergessen) [inline tests/lib/files/objectstore/noopscanner.php]: Will push as soon as jenkins replied

Comment (nickvergessen) [inline tests/lib/files/objectstore/noopscanner.php]: pushed

**Comment by PVince81** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**